### PR TITLE
Juniper Support for PowerMax

### PR DESCRIPTION
**Pull request by shanmydell**
# Description
Juniper Support for PowerMax

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/389 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/authorization/_index.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ The following diagram shows a high-level overview of CSM for Authorization with
 {{<table "table table-striped table-bordered table-sm">}}
 |               | PowerMax         | PowerFlex | PowerScale |
 |---------------|:----------------:|:-------------------:|:----------------:|
-| Storage Array |5978.479.479, 5978.711.711, Unisphere 9.2|    3.5.x, 3.6.x    | OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3 |
+| Storage Array |5978.479.479, 5978.711.711, Unisphere 10.0|    3.5.x, 3.6.x    | OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3 |
```
Comment (mjsdell) [inline content/docs/authorization/_index.md:46]: If we have customers on 9.x, what are we telling them?

Comment (prablr79) [reply content/docs/authorization/_index.md:46]: we should add both 9.x and 10.x

**Review by boyamurthy — COMMENTED** [review]
Please add uCode version for Juniper arrays as well .

File: content/docs/authorization/_index.md (hunk 1 above)
Comment (shanmydell) [reply content/docs/authorization/_index.md:46]: We are mandating them to upgrade to 10.x

**Review by rajendraindukuri — APPROVED** [review]

**Review by boyamurthy — APPROVED** [review]

**Pull request merged**